Missing tensorflow_cc when building with DeePMD

### Details

I'm trying to build DeePMD-kit in my Abacus using
`cmake -B build -DLibxc_DIR=~/libxc-6.2.2 -DENABLE_DEEPKS=1 -DTorch_DIR=~/libtorch/share/cmake/Torch/ -Dlibnpy_INCLUDE_DIR=~/libnpy -DDeePMD_DIR=~/deepmd-kit -DTensorFlow_DIR=~/.local/lib/python3.10/site-packages/tensorflow/ -DENABLE_LIBRI=ON -DUSE_CUDA=1 -DCMAKE_CUDA_COMPILER=/usr/local/cuda-11.8/bin/nvcc`

There is libtensorflow_cc.so inside the Python library, but still cannot be found:

`CMake Error at /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find TensorFlow (missing: tensorflow_cc)
Call Stack (most recent call first):
  /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindTensorFlow.cmake:18 (find_package_handle_standard_args)
  CMakeLists.txt:586 (find_package)`

But in installing DeePMD-kit it made sense using Python library. Is there a mistake in finding TensorFlow?

### Have you read FAQ on the online manual http://abacus.deepmodeling.com/en/latest/community/faq.html

- [X] Yes, I have read the FAQ part on online manual.

### Task list for Issue attackers (only for developers)

- [ ] Understand the problem or question described by the user.
- [ ] Check if the issue is a known problem or has been addressed in the documentation.
- [ ] Test the issue or problem on a similar system or environment, if possible.
- [ ] Identify the root cause or provide clarification on the user's question.
- [ ] Provide a step-by-step guide, including any necessary resources, to resolve the issue or answer the question.
- [ ] If the issue is related to documentation, update the documentation to prevent future confusion (optional).
- [ ] If the issue is related to code, consider implementing a fix or improvement (optional).
- [ ] Review and incorporate any relevant feedback from users or developers.
- [ ] Ensure the user's issue is resolved or their question is answered and close the ticket.

--- a/cmake/FindTensorFlow.cmake
+++ b/cmake/FindTensorFlow.cmake
@@ -7,7 +7,7 @@
 
 
 find_library(tensorflow_cc
-    NAMES tensorflow_cc
+    NAMES tensorflow_cc libtensorflow_cc.so.2
     HINTS ${TensorFlow_DIR}
     PATH_SUFFIXES "lib"
     )

